### DISCUSSION TITLE
스크린샷(셀카기능) 관련 오류질문입니다.

### DISCUSSION THREAD

**Discussion by hjs6082** [naverz/zepeto-studio-kor#1124] [5. Q&A]
셀카기능을 키면 오류가 생깁니다.
![image](https://user-images.githubusercontent.com/70927848/182752718-cdf1cb02-97a3-4e90-9747-9ea68e670a59.png)
보시다시시피 셀카봉이 밑으로가거나 이상한 곳으로 갑니다.. 셀카모드 말고는 카메라모드, 찍기 정상작동합니다. 스크립트도 모두 넣은것을 확인했습니다.

![image](https://user-images.githubusercontent.com/70927848/182752801-cfe766e6-306f-40df-8a25-5938a2e769b6.png)

![image](https://user-images.githubusercontent.com/70927848/182752877-74307c97-a506-4730-be62-ea17aaa32cfd.png)
가린곳은 스크립트 위치입니다. 왜 이런 현상이 발생할까요..?

두번째 질문은 
![image](https://user-images.githubusercontent.com/70927848/182753041-80961987-6202-4cb0-8f0f-cc6541220126.png)
이 IK Pass 위치입니다. 
![image](https://user-images.githubusercontent.com/70927848/182753130-ded6177b-5bd0-4f5a-8bda-5908e9665e37.png)
밑에 아무것도 뜨지않는데 혹시 어디위치에있을까요..?

Q1. 이런 현상이 왜 발생할까요? 
Q2. IK PASS 키는 법

**Comment by hjs6082** [comment]
IK PASS는 찾았습니다.. 그런데 Q1현상이 지속되네요.. Q1현상만 답변부탁드립니다!!

**Reply by Zepeto-TechEvangelist** [reply]
안녕하세요, https://studio.zepeto.me/guides/screenshot 가이드 하단에 있는 셀피 카메라 설정을 적용하셨는지 확인 부탁 드립니다.

**Reply by hjs6082** [reply]
> 안녕하세요, https://studio.zepeto.me/guides/screenshot 가이드 하단에 있는 셀피 카메라 설정을 적용하셨는지 확인 부탁 드립니다.

있는 그대로 가져왔고, 설정도 동일합니다. 
![image](https://user-images.githubusercontent.com/70927848/182758244-9675e248-879f-4022-aab4-8094e366d665.png)

![image](https://user-images.githubusercontent.com/70927848/182758285-99453805-e2bd-4bad-9a97-6647e6e7fb43.png)

**Reply by hjs6082** [reply]
건드린거 자체가 없고, 그냥 임포트만 시켰을뿐인데 에러가 뜹니다..  IK PASS는 켜주었고요.

**Comment by hjs6082** [comment]
해결했습니다!  프로젝트 세팅에서 인풋매니저를 Both로 바꾸는것으로 해결했습니다
